index/templates/index/base.html

```diff
@@ -192,8 +192,8 @@
     <div class="footer sticky-footer">
       <a href="" class="p11" style="position:relative;left:30px;">Contact us</a>
       <p class="p10" style="position:relative;left:400px;">
-      © NTHU Online Judge 2014, Templates are from Bootstrap Project</p> 
-      <p id='time' 
+      © NTHU Online Judge 2014, Templates are from Bootstrap Project</p>
+      <p id='time'
       style="font-size: 18px;color:white;position:relative;float:right;right:30px;">
```

Comment:
could you please refactor this with .css file.
html should be as clear as enough